source/common/json/config_schemas.cc

```diff
@@ -76,14 +76,131 @@ const std::string Json::Schema::HTTP_CONN_NETWORK_FILTER_SCHEMA(R"EOF(
   {
     "$schema": "http://json-schema.org/schema#",
     "definitions" : {
-      "access_log": {
+      "status_code" : {
         "type" : "object",
         "properties" : {
-          "path" : {"type" : "string"},
-          "format" : {"type" : "string"},
-          "filter" : {"type" : "object"}
+          "type" : {
+            "type" : "string",
+            "enum" : ["status_code"]
+          },
+          "op" : {
+            "type" : "string",
+            "enum" : [">=", "="]
+          },
+          "value" : {
+            "type" : "integer",
+            "minimum" : 0,
+            "maximum" : 599
+          },
+          "runtime_key" : {"type" : "string"}
+        },
+        "required" : ["type", "op", "value"],
+        "additionalProperties" : false
+      },
+      "duration" : {
+        "type" : "object",
+        "properties" : {
+          "type" : {
+            "type" : "string",
+            "enum" : ["duration"]
+          },
+          "op" : {
+            "type" : "string",
+            "enum" : [">=", "="]
+          },
+          "value" : {
+            "type" : "integer",
+            "minimum" : 0
+          },
+          "runtime_key" : {"type" : "string"}
         },
-        "required" : ["path"],
+        "required" : ["type", "op", "value"],
+        "additionalProperties" : false
+      },
+      "not_healthcheck" : {
+        "type" : "object",
+        "properties" : {
+          "type" : {
+            "type" : "string",
+            "enum" : ["not_healthcheck"]
+          }
+        },
+        "required" : ["type"],
+        "additionalProperties" : false
+      },
+      "traceable_request" : {
+        "type" : "object",
+        "properties" : {
+          "type" : {
+            "type" : "string",
+            "enum" : ["traceable_request"]
+          }
+        },
+        "required" : ["type"],
+        "additionalProperties" : false
+      },
+      "runtime" : {
+        "type" : "object",
+        "properties" : {
+          "type" : {
+            "type" : "string",
+            "enum" : ["runtime"]
+          },
+          "key" : {"type" : "string"}
+        },
+        "required" : ["type", "key"],
+        "additionalProperties" : false
+      },
+      "logical_and" : {
+        "type" : "object",
+        "properties" : {
+          "type" : {
+            "type" : "string",
+            "enum" : ["logical_and"]
+          },
+          "filters" : {
```

Comment:
Yes, good call. I didn't think about putting those two together at the time. Thank you!